app/models/preference_sections/map_section.rb

```diff
@@ -0,0 +1,15 @@
+module PreferenceSections
+  class MapSection
```

Comment:
Note: the preferences are hard to find in the config section. We could move them to a new map settings page.